### PR TITLE
HPCC-14132 Fix ssh calls from scripts to include LogLevel=QUIET

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner Michael.Gardner@lexisnexis.com

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@xwang2713 Please review when you get a chance

File: initfiles/componentfiles/thor/sshslaves [hunk 1]
```diff
@@ -22,7 +22,7 @@ COUNTER=1
 for slave in $(cat slaves); do
     echo sshing slave $COUNTER - $slave:
     echo ssh $THORUSER@$slave $1
-    ssh $THORUSER@$slave $1
+    ssh -o LogLevel=QUIET $THORUSER@$slave $1
```
Comment (mckellyln) [inline initfiles/componentfiles/thor/sshslaves]: Should BatchMode and StrictHostChecking also be set here ?

**Comment by mckellyln** [comment]
What about the ssh cmds in:
esp/services/ws_machine/ws_machineService.cpp
esp/services/ws_machine/ws_machineServiceRexec.cpp
Should these also get these options ?

**Comment by Michael-Gardner** [comment]
https://track.hpccsystems.com/browse/HPCC-14153  the ws_machineService stuff is getting handled separately because I need to have Kevin review those changes.  So they won't be in this PR.  Although I already have ws_machineService.cpp taken care of.  It now uses CFRunSSH, I just need to do ws_machineServiceRexec.cpp now.

**Comment by xwang2713** [comment]
Looks good

**Pull request merged**